Task_3.py

```diff
@@ -1,56 +1,38 @@
 from source import get_data
-import csv
-import os
-from operator import itemgetter
 import re
+from document_creation import file_create
+from operator import itemgetter
 
-# 3) Task_3.py: to create file words-stats1.cvs и words-stats2.cvs
-# containing information about the most popular words from positive and negative messages.
-def word_list_from_positive_comment(main_dict):
-    positive_comment = []
+def message_list(main_dict):
+    '''function returns positive and negative messages list'''
+    positive_comments = []
+    negative_comments = []
     for item in main_dict.values():
         if 'overall' in item.keys():
-            if item['overall'] >= 3:
-                positive_comment.append(item['reviewText'])
-    positive_comment = str(positive_comment).replace("'",'')
-    word_positive_comment = []
-    for item in re.split(r'\W+', positive_comment):
-        if len(item) > 1:
-            word_positive_comment.append(item.lower())
-    popular_in_positive_comment = dict.fromkeys(word_positive_comment, 0)
-    for word in word_positive_comment:
-        popular_in_positive_comment[word] = popular_in_positive_comment.get(word) + 1
-    return popular_in_positive_comment
+            if item['overall'] >= 2.5:
```

Comment:
What about this:
```
words = re.split(r'\W+', item['reviewText'].replace("'", '_').lower())
(positive_comments if item['overall'] >= 2.5 else negative_comments).extend(words)
```
?

Reply:
I found only such variant.
`def get_text(main_dict, overall_values):
    '''function gets a text and returns list of reviews in depends on overall values'''
    message_list = [item['reviewText'] for item in main_dict.values()
                    if 'overall' in item.keys() and item['overall'] in overall_values]
    return message_list

def split_text_up(text):
    '''function gets a text and returns list of words from this text'''
    list_of_words = []
    for review in text:
        list_of_words.extend(re.split(r'\W+', review.replace("'", '_').lower()))
    return list_of_words`

Reply:
I worry about this part:
```
   list_of_words = []
    for review in text:
        list_of_words.extend(re.split(r'\W+', review.replace("'", '_').lower()))
```
but I really don't know how do it simpler, cause I need result like ['w1', 'w2',...., 'wn']
and when I write such code 
`list_of_words = [word for word in [re.split(r'\W+', review.replace("'", '_').lower()) for review in text]]`
I get the result like [ ['w1', 'w2', ....] , [...], ....., ['wn', 'wm', ...] ]